src/components/inline/InputText/InputText.js

```diff
@@ -0,0 +1,32 @@
+import PropTypes from "prop-types";
+import React from "react";
+
+import s from "./InputText.module.css";
+
+export const InputTextPropType = PropTypes.shape({
+  placeholderText: PropTypes.string,
+  widthDesktop: PropTypes.string.isRequired,
+  widthMobile: PropTypes.string.isRequired,
+  onChange: PropTypes.func,
+  value: PropTypes.string,
+});
+
+export const InputText = ({ inputText }) => {
+  return (
+    <input
+      className={s.inputText}
+      style={{
+        "--input-text-desktop": `${inputText.widthDesktop}`,
+        "--input-text-mobile": `${inputText.widthMobile}`,
+      }}
+      type="text"
```

Comment:
You might actually want this to be a prop, since there are other text-like types that provide extra validation, such as  [`type="email"`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/email).

Reply:
ok